pyswarms/backend/topology/random.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+
+"""
+A Random Network Topology
+
+This class implements a random topology. All particles are connected in a random fashion.
```

Comment:
It would be nice if we add a short explanation on why `dijkstra` is used and how it relates to the `Random` topology. :smile: